src/main/java/org/gridsuite/modification/server/modifications/BatteryCreation.java

```diff
@@ -219,8 +219,9 @@ private void createBatteryActivePowerControl(BatteryCreationInfos batteryCreatio
             } catch (PowsyblException e) {
                 activePowerRegulationReports.add(Report.builder()
                         .withKey("ActivePowerExtensionAddError")
-                        .withDefaultMessage("cannot add active power extension on battery with id=${id} : " + e.getMessage())
+                        .withDefaultMessage("cannot add active power extension on battery with id=${id} : ${message}")
                         .withValue("id", batteryCreationInfos.getEquipmentId())
+                        .withValue("message", e.getMessage())
```

Comment:
I think as soon as you have a parameter ({id} here) in the message, it is ok

Reply:
Every variable needs to be in the .withValue clause